src/Models/Assessment/Datasets/BusinessCaseDataset.cs

```diff
@@ -6,17 +6,23 @@ public BusinessCaseDataset()
         {
             OnPremIaaSCostDetails = new BusinessCaseDatasetCostDetails();
             OnPremPaaSCostDetails = new BusinessCaseDatasetCostDetails();
+            OnPremAvsCostDetails = new BusinessCaseDatasetCostDetails();
             AzureIaaSCostDetails = new BusinessCaseDatasetCostDetails();
             AzurePaaSCostDetails = new BusinessCaseDatasetCostDetails();
+            AzureAvsCostDetails = new BusinessCaseDatasetCostDetails();
             TotalYOYCashFlows = new BusinessCaseYOYCostDetailsJSON();
             IaaSYOYCashFlows = new BusinessCaseYOYCostDetailsJSON();
+            AvsYOYCashFlows = new BusinessCaseYOYCostDetailsJSON();
         }
 
-        public BusinessCaseDatasetCostDetails OnPremIaaSCostDetails { get; set; }
-        public BusinessCaseDatasetCostDetails OnPremPaaSCostDetails { get; set; }
-        public BusinessCaseDatasetCostDetails AzureIaaSCostDetails { get; set; }
-        public BusinessCaseDatasetCostDetails AzurePaaSCostDetails { get; set; }
-        public BusinessCaseYOYCostDetailsJSON TotalYOYCashFlows { get; set; }
-        public BusinessCaseYOYCostDetailsJSON IaaSYOYCashFlows { get; set; }
+        public BusinessCaseDatasetCostDetails OnPremIaaSCostDetails { get; set; } = null;
+        public BusinessCaseDatasetCostDetails OnPremPaaSCostDetails { get; set; } = null;
+        public BusinessCaseDatasetCostDetails OnPremAvsCostDetails { get; set; } = null;
+        public BusinessCaseDatasetCostDetails AzureIaaSCostDetails { get; set; } = null;
+        public BusinessCaseDatasetCostDetails AzurePaaSCostDetails { get; set; } = null;
+        public BusinessCaseDatasetCostDetails AzureAvsCostDetails { get; set; } = null;
+        public BusinessCaseYOYCostDetailsJSON TotalYOYCashFlows { get; set; } = null;
+        public BusinessCaseYOYCostDetailsJSON IaaSYOYCashFlows { get; set; } = null;
+        public BusinessCaseYOYCostDetailsJSON AvsYOYCashFlows { get; set; } = null;

```

Comment:
make nullable

Reply:
This is another way of making the properties nullable when Feature 'nullable reference types' is not allowed in certain .NET versions like the one we are using